server/src/Config.hs

```diff
@@ -32,6 +32,8 @@ data Config = Config
   , configSyncIntervalMicroSeconds :: Maybe Int
   -- | Enable journaling, only in conjunction with periodic syncing
   , configEnableJournaling :: Bool
+  , configDisableSentryLogging :: Bool
+  , configSentryDSN :: Maybe String
```

Comment:
Could you add some haddock comments here explaining the relation of the two, and the reason why they exist?
I suppose the `configDisableSentryLogging` is useful for the case where a `SENTRY_DSN` environment variable is set, but we actually don't want to log to sentry.